apps/google-analytics-4/frontend/src/components/config-screen/api-access/display/DisplayServiceAccountCard.tsx

```diff
@@ -237,15 +238,17 @@ const DisplayServiceAccountCard = (props: Props) => {
       );
     }
 
-    return <Badge variant="positive">Active</Badge>;
+    return <Badge variant="positive">Access correctly configured</Badge>;
```

Comment:
whatever you do about my [comment above](https://github.com/contentful/apps/pull/3012/files#r1153818764) – the same applies to this spot, too. perhaps another variable. i almost want to put all of this copy into a JSON object at this point and treat it like we would i18n strings so we're DRY about it more easily

Reply:
I know that feel 😅 . For now I avoided refactoring entirely since there's a lot more that needs to be done where that came from!